python/lsst/ts/dimm/dimm_csc.py

```diff
@@ -271,7 +348,7 @@ async def health_monitor(self):
                         ),
                     )
 
-                    self.fault(code=TELEMETRY_LOOP_DONE, report=error_report)
+                    await self.fault(code=TELEMETRY_LOOP_DONE, report=error_report)
```

Comment:
Same as above, I think `fault` is a regular method not a coroutine.

Reply:
Just like above I'll leave it like this.

Reply:
since you fixed it, can you remove it now?

Reply:
Done